include/proj/datum.hpp

```diff
@@ -541,7 +541,7 @@ class PROJ_GCC_DLL RealizationMethod : public util::CodeList {
     PROJ_FRIEND_OPTIONAL(RealizationMethod);
     PROJ_DLL explicit RealizationMethod(
         const std::string &nameIn = std::string());
-    PROJ_DLL RealizationMethod(const RealizationMethod &other);
+    PROJ_DLL RealizationMethod(const RealizationMethod &other) = default;
```

Comment:
There's already in datum.cpp:
```
RealizationMethod::RealizationMethod(const RealizationMethod &) = default;
```
You might want to remove it then

Reply:
But it is removed from datum.cpp as well. Or am I missing something else?

Reply:
oh sorry I missed that!